geode-core/src/test/java/org/apache/geode/cache/query/internal/OrderByComparatorJUnitTest.java

```diff
@@ -173,36 +157,58 @@ public void testUnsupportedOrderByForPR() throws Exception {
 
   @Test
   public void testSupportedOrderByForRR() throws Exception {
-
     String unsupportedQueries[] =
-        {"select distinct p.status from /portfolio1 p order by p.status, p.ID",
-
-        };
+        {"select distinct p.status from /portfolio1 p order by p.status, p.ID"};
     Object r[][] = new Object[unsupportedQueries.length][2];
-    QueryService qs;
-    qs = CacheUtils.getQueryService();
     Position.resetCounter();
-    // Create Regions
 
+    // Create Regions
     Region r1 = CacheUtils.createRegion("portfolio1", Portfolio.class);
 
     for (int i = 0; i < 50; i++) {
       r1.put(new Portfolio(i), new Portfolio(i));
     }
 
     for (int i = 0; i < unsupportedQueries.length; i++) {
-      Query q = null;
-
+      Query q;
       CacheUtils.getLogger().info("Executing query: " + unsupportedQueries[i]);
       q = CacheUtils.getQueryService().newQuery(unsupportedQueries[i]);
       try {
         r[i][0] = q.execute();
-
       } catch (QueryInvalidException qe) {
         qe.printStackTrace();
         fail(qe.toString());
       }
     }
   }
 
+  /**
+   * Tests three cases that were not originally covered by the original tests in this class. (To
```

Comment:
This comment probably doesn't need to be a javadoc, and the note about coverage isn't needed. I'd make this comment say something like, "The following tests cover edge cases in OrderByComparator".